utils/modeling.py

```diff
@@ -255,7 +258,7 @@ def train(self): #, mlflow_logger):
         return self.model
 
 
-    def evaluate(self, splits=["train", "val"]):
+    def evaluate(self, splits=["train", "val"], constant_preprocessor=None):
```

Comment:
Sounds good! Would you also extend the docstring?

In case you would want to give a type hint, use `collections.abc.Callable`.